lib/generators/paper_trail/install/templates/paper_trail.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+# This file is automatically generated by `rails generate paper_trail:install`
+# and goes in `YourApp/config/initializers/`.
+# Each model can override most of these global settings on a per-model basis.
+
+## Enable/disable PaperTrail globally
+PaperTrail.config.enabled = true
+
+## Track versions on create, update, & destroy, but not touch events to reduce version table size.
+## Add `touch` below to track touch events, or comment out (default behaviour tracks all events).
+PaperTrail.config.has_paper_trail_defaults = {
+  on: %i[create update destroy]
```

Comment:
i like the generation of this file but i don't think the config generated here should differ from the existing defaults. i realize this goes against part of the intent of your pr but it results in an implicit change to the defaults for anyone starting from scratch which feels inconsistent to me.